docs/source/contributing.rst

```diff
@@ -15,16 +15,17 @@ Before Contributing
 
 Before sending us a Pull Request, please consider this:
 
-* Is the contribution entirely your own work, or is it already licensed under an LGPL 2.1 compatible Open Source License? If not, we unfortunately cannot accept it.
+* Is the contribution entirely your own work, or is it already licensed under an LGPL 2.1 compatible Open Source License? If not, we, unfortunately, cannot accept it.
 
-* Is the code adequately commented for people to understand how it is structured?
+* Is the code adequately commented on to people understand how it is structured?
```

Comment:
Again, this sentence is a bit awkward.

Suggestion:

Is the code adequately commented so others can understand how it is structured?